### ISSUE TITLE
Are code block smart-connections still working

### ISSUE THREAD

**Issue by aikruger** [brianpetro/obsidian-smart-connections#845]
I can't seem to get the ```smart-connections code block to work any more since installing the the latest version of smart connections. It gives me a blank page rather than an error. 
Thanks

**Comment by brianpetro** [comment]
Good catch. I can think of some reasons it wouldn't be working based on the latest update. Fortunately it should be a relatively easy fix. Thanks for bringing this to my attention 🌴

**Comment by brianpetro** [comment]
Should be fixed in `v2.2.83/2.3.25` 🌴

**Comment by BertTheDragon** [comment]
Please accept my apologies; I am suffering from the same error but unsure what to do next. Should I uninstall the recently installed v2.2.83 smart connection plugin and reinstall the v2.2.83/2.3.25 version? I need help locating the newer version (v2.2.83/2.3.25).

**Comment by brianpetro** [comment]
@BertTheDragon have you restarted Obsidian since updating to the latest version? Sometimes that fixes code-block issues 🌴

**Comment by BertTheDragon** [comment]
Yes, unfortunately that did not help. I have tried using several different embedding models (including the TayloAI/bge-micro-v2) to see if that resolved the problem, but unfortunately it did not.

**Comment by brianpetro** [comment]
There should be an error in the developer console logs, screenshot that and I'll take a look 🌴

**Comment by BertTheDragon** [comment]
<img width="557" alt="Screenshot 2024-10-25 at 00 12 03" src="https://github.com/user-attachments/assets/35933471-56e1-43d9-a942-7f2783814f03">

Do you need more?

**Comment by BertTheDragon** [comment]
<img width="557" alt="Screenshot 2024-10-25 at 00 22 21" src="https://github.com/user-attachments/assets/32819ee2-c323-48dc-ba17-f2d417cd6303">

**Comment by brianpetro** [comment]
Do any errors appear when you try to add a new code block? 🌴

**Comment by BertTheDragon** [comment]
FYI this morning it has taken 5 hours to complete 50% of my vaults embedding using TaylorAI/bge-micro-v2 and smart connections v2.2.83. Embedding took seconds using the previous smart connections version.

**Comment by huachuman** [comment]
brian just scrap v2 and start fresh with v1 again
